FirebaseCore/Internal/Sources/HeartbeatLogging/HeartbeatStorage.swift

```diff
@@ -51,20 +51,30 @@ final class HeartbeatStorage: HeartbeatStorageProtocol {
   // MARK: - Instance Management
 
   /// Statically allocated cache of `HeartbeatStorage` instances keyed by string IDs.
-  private static var cachedInstances: [String: WeakContainer<HeartbeatStorage>] = [:]
+  private nonisolated(unsafe) static var cachedInstances: [
```

Comment:
Add a comment about why this is `nonisolated(unsafe)`

Reply:
Done.